resources/hotwater_appliances.rb

```diff
@@ -28,7 +28,7 @@ def self.apply(model, runner, weather, living_space,
 
     # Schedules init
     timestep_minutes = (60.0 / model.getTimestep.numberOfTimestepsPerHour).to_i
-    start_date = model.getYearDescription.makeDate(1, 1)
+    start_date = OpenStudio::Date.new(OpenStudio::MonthOfYear.new(1), 1, model.getYearDescription.assumedYear)
```

Comment:
Does this get rid of the OS warning? Yay!

Reply:
Yes.